src/basic/parse-util.c

```diff
@@ -332,9 +333,15 @@ int parse_syscall_and_errno(const char *in, char **name, int *error) {
 
         p = strchr(in, ':');
         if (p) {
-                e = parse_errno(p + 1);
-                if (e < 0)
-                        return e;
+                if (streq(p + 1, "kill"))
```

Comment:
maybe do a `p++` before this, instead of adding one in each streq() separately

Reply:
done